scene/gui/view_panner.cpp

```diff
@@ -39,39 +39,43 @@ bool ViewPanner::gui_input(const Ref<InputEvent> &p_event, Rect2 p_canvas_rect)
 	if (mb.is_valid()) {
 		Vector2 scroll_vec = Vector2((mb->get_button_index() == MouseButton::WHEEL_RIGHT) - (mb->get_button_index() == MouseButton::WHEEL_LEFT), (mb->get_button_index() == MouseButton::WHEEL_DOWN) - (mb->get_button_index() == MouseButton::WHEEL_UP));
 		if (scroll_vec != Vector2()) {
```

Comment:
```suggestion
		if (scroll_vec != Vector2() && mb->is_pressed()) {
```
You could just do this instead of indenting everything.

Reply:
Merged the two if statements.